lib/activerecord-clean-db-structure/clean_dump.rb

```diff
@@ -30,9 +30,9 @@ def run
       # Reduce noise for id fields by making them SERIAL instead of integer+sequence stuff
       #
       # This is a bit optimistic, but works as long as you don't have an id field thats not a sequence/uuid
-      dump.gsub!(/^    id integer NOT NULL,$/, '    id SERIAL PRIMARY KEY,')
```

Comment:
I wonder if we shouldn't just make the `,` optional instead?

i.e. make it `/^    id integer NOT NULL,?$/`